### PR TITLE
Part 2: use ModSpec definitions list syntax (#502)

### LINKED ISSUE #502: Wrong Templates used for Requirements, Recommendations and Permissions
Wrong template used for all requirements, recommendations and permissions.

### PR DESCRIPTION
**Pull request by tomkralidis**
Fixes #502 

cc @ShaneMill1

### CODE REVIEW

File: extensions/pubsub/standard/abstract_tests/pubsub-message-payload/ATS_test_id.adoc
Comment (m-burgoyne) [inline extensions/pubsub/standard/abstract_tests/pubsub-message-payload/ATS_test_id.adoc:1]: Should this refer to AysncAPI 3.0 rather than AysncAPI 2.0?

Comment (m-burgoyne) [inline extensions/pubsub/standard/abstract_tests/pubsub-message-payload/ATS_test_id.adoc:1]: Should this refer to AysncAPI 3.0 rather than AysncAPI 2.0?

**Review by m-burgoyne — APPROVED** [review]
Looks good, updated the references from AsyncAPI 2.0 to AsyncAPI 3.0

**Review by chris-little — APPROVED** [review]
Tedious, but ;ooks good!

**Comment by chris-little** [comment]
As agreed at EDR API SWG Meeting 111

**Pull request merged**